pywinauto/application.py

```diff
@@ -847,7 +847,19 @@ def __connect(self, **kwargs):
     Connect_ = __connect  # A deprecated name. Should be removed in 0.6.X
 
     def connect(self, **kwargs):
-        """Connects to an already running process"""
+        """Connects to an already running process
+
+        The action is performed according to only one of parameters
+
+        :param process: a process ID of the target
+        :param handle: a window handle of the target
+        :param path: a path used to launch the target
+
+        .. seealso::
+
+           :func:`pywinauto.findwindows.find_elements` - the keyword arguments that
+           are also can be used instead of **process**, **handle** or **path**

```

Comment:
I would say `instead of or in addition to`.
Example: `app.connect(path='explorer.exe', title='Program Files')`